safe_transaction_service/history/serializers.py

```diff
@@ -258,15 +259,33 @@ class SafeDelegateDeleteSerializer(serializers.Serializer):
     delegate = EthereumAddressField()
     signature = HexadecimalField(min_length=65)
 
-    def check_signature(self, ethereum_client: EthereumClient, safe_address: str,
-                        signature: bytes, operation_hash: bytes, safe_owners: List[str]) -> Optional[str]:
+    def get_safe_owners(self, ethereum_client: EthereumClient, safe_address: ChecksumAddress) -> List[ChecksumAddress]:
+        safe = Safe(safe_address, ethereum_client)
+        try:
+            return safe.retrieve_owners(block_identifier='pending')
+        except BadFunctionCallOutput:  # Error using pending block identifier
+            return safe.retrieve_owners(block_identifier='latest')
+
+    def get_valid_delegators(self, ethereum_client: EthereumClient, safe_address: ChecksumAddress,
+                             delegate: ChecksumAddress) -> List[ChecksumAddress]:
+        """
+        :param ethereum_client:
+        :param safe_address:
+        :param delegate:
+        :return: Valid delegators for a Safe. A delegate should be able to remove itself
+        """
+        return self.get_safe_owners(ethereum_client, safe_address) + [delegate]
```

Comment:
Are owners considered `delegates` in this method?

Reply:
No, owners are delegators